docs/UserGuide.md

```diff
@@ -67,21 +67,22 @@ Adds a new friend to Amigos. A friend has:
 * *(optional)* email
 * *(optional)* address
 * *(optional)* description
+* *(optional)* tags
 
-**Format**: `addfriend n/[NAME]  p/[PHONE_NUMBER] e/[EMAIL]  a/[ADDRESS] d/[DESCRIPTION]`
+**Format**: `addfriend n/[NAME]  p/[PHONE_NUMBER] e/[EMAIL]  a/[ADDRESS] d/[DESCRIPTION] [t/TAG]…`
```

Comment:
This formatting does not conform to the UG command formatting, please refer to the table in the UG. e.g. [] is only for optional params.